### PR TITLE
WGS pipeline updates

### PR DESCRIPTION
**Pull request by jkokosar**
- Revert CPU/memory settings for WGS pre-processing step
- make trimmomatic step optional in the WGS workflow

### CODE REVIEW

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -16,8 +16,9 @@ Added
 
 Changed
 -------
-- Optimize CPU and memory usage in processes ``wgs-preprocess`` and
```
Comment (AnzeLovse) [inline docs/CHANGELOG.rst:19]: Why is this entry removed?

File: resolwe_bio/processes/workflows/wgs_gvcf.py [hunk 2]
```diff
@@ -61,33 +61,22 @@ class Input:
...
+                default=False,
+            )
+
             adapters = DataField(
                 "seq:nucleotide",
                 label="Adapter sequences",
                 required=False,
                 description="Adapter sequences in FASTA format that will "
                 "be removed from the reads.",
+                disabled="!trimming_options.enable_trimming",
```
Comment (AnzeLovse) [inline resolwe_bio/processes/workflows/wgs_gvcf.py:79]: I am not sure if `disabled`  parameter is working as expected. Could you please check if fields are disabled?

File: resolwe_bio/processes/workflows/wgs_gvcf.py [hunk 3]
```diff
@@ -359,17 +371,16 @@ def run(self, inputs, outputs):
...
-
+        multiqc_inputs = [
+            inputs.reads,
+            bam,
+            summary,
+            wgs_metrics,
+            insert_size,
+        ]
+        if inputs.trimming_options.enable_trimming:
+            multiqc_inputs.append("trimmomatic")
```
Comment (AnzeLovse) [inline resolwe_bio/processes/workflows/wgs_gvcf.py:382]: `trimmomatic` should be a variable and not a string.

**Review by AnzeLovse — COMMENTED** [review]
Please consider my comments.

File: resolwe_bio/processes/workflows/wgs_gvcf.py (hunk 2 above)
Comment (jkokosar) [reply resolwe_bio/processes/workflows/wgs_gvcf.py:79]: The fields work as expected!

File: resolwe_bio/processes/workflows/wgs_gvcf.py (hunk 3 above)
Comment (jkokosar) [reply resolwe_bio/processes/workflows/wgs_gvcf.py]: Fixed, thank you!

File: resolwe_bio/processes/workflows/wgs_gvcf.py [hunk 4]
```diff
@@ -138,19 +127,40 @@ class Trimming:
...
                 required=False,
                 description="Drop the read if it is below a specified length.",
+                disabled="!trimming_options.enable_trimming",
+            )
+
+        class GatkOptions:
+            """Options."""
+
+            intervals = DataField(
+                "bed",
```
Comment (romunov) [inline resolwe_bio/processes/workflows/wgs_gvcf.py:152]: Argument `data_type` could be listed explicitly as the rest of the arguments. For example

```
data_type="bed",
```

**Review by romunov — APPROVED** [review]
Looks good.

**Pull request merged**